transports/janus_websockets.c

```diff
@@ -0,0 +1,906 @@
+		case LWS_CALLBACK_CLOSED:
+			return "LWS_CALLBACK_CLOSED";
+		case LWS_CALLBACK_CLOSED_HTTP:
+			return "LWS_CALLBACK_CLOSED_HTTP";
+		case LWS_CALLBACK_RECEIVE:
+			return "LWS_CALLBACK_RECEIVE";
+		case LWS_CALLBACK_CLIENT_RECEIVE:
+			return "LWS_CALLBACK_CLIENT_RECEIVE";
+		case LWS_CALLBACK_CLIENT_RECEIVE_PONG:
+			return "LWS_CALLBACK_CLIENT_RECEIVE_PONG";
+		case LWS_CALLBACK_CLIENT_WRITEABLE:
+			return "LWS_CALLBACK_CLIENT_WRITEABLE";
+		case LWS_CALLBACK_SERVER_WRITEABLE:
+			return "LWS_CALLBACK_SERVER_WRITEABLE";
+		case LWS_CALLBACK_HTTP:
+			return "LWS_CALLBACK_HTTP";
+		case LWS_CALLBACK_HTTP_BODY:
+			return "LWS_CALLBACK_HTTP_BODY";
+		case LWS_CALLBACK_HTTP_BODY_COMPLETION:
+			return "LWS_CALLBACK_HTTP_BODY_COMPLETION";
+		case LWS_CALLBACK_HTTP_FILE_COMPLETION:
+			return "LWS_CALLBACK_HTTP_FILE_COMPLETION";
+		case LWS_CALLBACK_HTTP_WRITEABLE:
+			return "LWS_CALLBACK_HTTP_WRITEABLE";
+		case LWS_CALLBACK_FILTER_NETWORK_CONNECTION:
+			return "LWS_CALLBACK_FILTER_NETWORK_CONNECTION";
+		case LWS_CALLBACK_FILTER_HTTP_CONNECTION:
+			return "LWS_CALLBACK_FILTER_HTTP_CONNECTION";
+		case LWS_CALLBACK_SERVER_NEW_CLIENT_INSTANTIATED:
+			return "LWS_CALLBACK_SERVER_NEW_CLIENT_INSTANTIATED";
+		case LWS_CALLBACK_FILTER_PROTOCOL_CONNECTION:
+			return "LWS_CALLBACK_FILTER_PROTOCOL_CONNECTION";
+		case LWS_CALLBACK_OPENSSL_LOAD_EXTRA_CLIENT_VERIFY_CERTS:
+			return "LWS_CALLBACK_OPENSSL_LOAD_EXTRA_CLIENT_VERIFY_CERTS";
+		case LWS_CALLBACK_OPENSSL_LOAD_EXTRA_SERVER_VERIFY_CERTS:
+			return "LWS_CALLBACK_OPENSSL_LOAD_EXTRA_SERVER_VERIFY_CERTS";
+		case LWS_CALLBACK_OPENSSL_PERFORM_CLIENT_CERT_VERIFICATION:
+			return "LWS_CALLBACK_OPENSSL_PERFORM_CLIENT_CERT_VERIFICATION";
+		case LWS_CALLBACK_CLIENT_APPEND_HANDSHAKE_HEADER:
+			return "LWS_CALLBACK_CLIENT_APPEND_HANDSHAKE_HEADER";
+		case LWS_CALLBACK_CONFIRM_EXTENSION_OKAY:
+			return "LWS_CALLBACK_CONFIRM_EXTENSION_OKAY";
+		case LWS_CALLBACK_CLIENT_CONFIRM_EXTENSION_SUPPORTED:
+			return "LWS_CALLBACK_CLIENT_CONFIRM_EXTENSION_SUPPORTED";
+		case LWS_CALLBACK_PROTOCOL_INIT:
+			return "LWS_CALLBACK_PROTOCOL_INIT";
+		case LWS_CALLBACK_PROTOCOL_DESTROY:
+			return "LWS_CALLBACK_PROTOCOL_DESTROY";
+		case LWS_CALLBACK_WSI_CREATE:
+			return "LWS_CALLBACK_WSI_CREATE";
+		case LWS_CALLBACK_WSI_DESTROY:
+			return "LWS_CALLBACK_WSI_DESTROY";
+		case LWS_CALLBACK_GET_THREAD_ID:
+			return "LWS_CALLBACK_GET_THREAD_ID";
+		case LWS_CALLBACK_ADD_POLL_FD:
+			return "LWS_CALLBACK_ADD_POLL_FD";
+		case LWS_CALLBACK_DEL_POLL_FD:
+			return "LWS_CALLBACK_DEL_POLL_FD";
+		case LWS_CALLBACK_CHANGE_MODE_POLL_FD:
+			return "LWS_CALLBACK_CHANGE_MODE_POLL_FD";
+		case LWS_CALLBACK_LOCK_POLL:
+			return "LWS_CALLBACK_LOCK_POLL";
+		case LWS_CALLBACK_UNLOCK_POLL:
+			return "LWS_CALLBACK_UNLOCK_POLL";
+		case LWS_CALLBACK_OPENSSL_CONTEXT_REQUIRES_PRIVATE_KEY:
+			return "LWS_CALLBACK_OPENSSL_CONTEXT_REQUIRES_PRIVATE_KEY";
+		case LWS_CALLBACK_USER:
+			return "LWS_CALLBACK_USER = 1000";
```

Comment:
Probably a leftover from some experiments, I'll remove it: thanks for spotting this.